_docs-2/getting-started/quickstart.md

```diff
@@ -90,121 +108,147 @@ The [accumulo-env.sh] file sets up environment variables needed by Accumulo:
-   prompt you for the password of `auth.principal` at login.
+2. Pick an authentication type and set {% plink -c auth.type %} accordingly. The
+   most common `auth.type` is `password` which requires {% plink -c auth.principal %}
+   to be set and {% plink -c auth.token %} to be set the password of
+   `auth.principal`. For the Accumulo shell, `auth.token` can be commented out
+   and the shell will prompt you for the password of `auth.principal` at login.
 
 ## Initialization
 
-Accumulo needs to initialize the locations where it stores data in Zookeeper
-and HDFS.  The following command will do this.
+Accumulo needs to initialize the locations where it stores data in Zookeeper and
+HDFS. The following command will do this.
 
-    accumulo init
+```
+accumulo init
+```
 
 The initialization command will prompt for the following information.
 
  * **Instance name** : This is the name of the Accumulo instance and its
    Accumulo clients need to know it in order to connect.
  * **Root password** : Initialization sets up an initial Accumulo root user and
-   prompts for its password.  This information will be needed to later connect
-   to Accumulo.
+   prompts for its password. This information will be needed to later connect to
+   Accumulo.
 
 ## Run Accumulo
 
 There are several methods for running Accumulo:
 
-1. Run Accumulo processes using `accumulo` command which runs processes in foreground and
-   will not redirect stderr/stdout. Useful for creating init.d scripts that run Accumulo.
+1. Run Accumulo processes using `accumulo` command which runs processes in
+   foreground and will not redirect stderr/stdout. Useful for creating init.d
+   scripts that run Accumulo.
 
-2. Run Accumulo processes as services using `accumulo-service` which uses `accumulo`
-   command but backgrounds processes, redirects stderr/stdout and manages pid files.
-   Useful if you are using a cluster management tool (i.e. Ansible, Salt, etc).
+2. Run individual Accumulo processes as services using `accumulo-service` which
+   uses `accumulo` command but backgrounds processes, redirects stderr/stdout
+   and manages pid files. This is useful if you are using a cluster management
+   tool (i.e. Ansible, Salt, etc).
 
 2. Run an Accumulo cluster on one or more nodes using `accumulo-cluster` (which
    uses `accumulo-service` to run services). Useful for local development and
-   testing or if you are not using a cluster management tool in production.
+   testing or if you are not using your own cluster management tool in
+   production.
 
 Each method above has instructions below.
 
-### Run Accumulo processes
+### Run individual Accumulo processes
 
-Start Accumulo processes (tserver, master, monitor, etc) using command below:
+Start Accumulo processes (tserver, master, monitor, etc) using the accumulo
+command followed by the service name. For example, to start only the tserver,
```

Comment:
Suggestion:

command followed by the service name.  For a fully operational instance, each individual service will need to be started.  For example, to start only a tserver

Reply:
Done in b145c875